<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tw_simulacion">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Principal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>609</width>
                <height>96</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>96</height>
               </size>
              </property>
              <property name="title">
               <string>Variables de entrada ascensor</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>21</y>
                 <width>504</width>
                 <height>56</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="lb_cantidadAscensores">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Cantidad ascensores</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_cantidadAscensores">
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="lb_tiempodeArranque">
                      <property name="text">
                       <string>Tiempo de arranque</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_tiempodeArranque">
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="lb_cantidadPisos">
                      <property name="text">
                       <string>Cantidad de pisos</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_cantidadPisos">
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="lb_capacidadAscensores">
                      <property name="text">
                       <string>Capacidad ascensores</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_capacidadAscensores">
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="lb_desplazamientoentrePisos">
                      <property name="text">
                       <string>Desplazamiento entre pisos</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_desplazamientoentrePisos">
                      <property name="minimumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>61</width>
                        <height>20</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="minimumSize">
               <size>
                <width>609</width>
                <height>95</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>95</height>
               </size>
              </property>
              <property name="title">
               <string>Variables de entrada persona</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>20</y>
                 <width>207</width>
                 <height>52</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="lb_tEntradaAscensor">
                    <property name="text">
                     <string>Tiempo de entrada ascensor</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sb_tEntradaAscensor">
                    <property name="minimumSize">
                     <size>
                      <width>61</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>61</width>
                      <height>20</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="lb_tSalidaAscensor">
                    <property name="text">
                     <string>Tiempo de salida ascensor</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sb_tSalidaAscensor">
                    <property name="minimumSize">
                     <size>
                      <width>61</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>61</width>
                      <height>20</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="minimumSize">
               <size>
                <width>609</width>
                <height>96</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>96</height>
               </size>
              </property>
              <property name="title">
               <string>Variables de entrada simulacion</string>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>40</y>
                 <width>192</width>
                 <height>22</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="lb_cantidadSimulacion">
                  <property name="text">
                   <string>Cantidad de simulaciones </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="sb_cantidadSimulacion">
                  <property name="minimumSize">
                   <size>
                    <width>61</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item alignment="Qt::AlignRight|Qt::AlignTop">
             <widget class="QPushButton" name="pb_limpiarTodo">
              <property name="text">
               <string>Limpiar todo</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft|Qt::AlignTop">
             <widget class="QPushButton" name="pb_iniciarSimulacion">
              <property name="text">
               <string>Iniciar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
